.buildkite/scripts/run-accessibility-check.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e
+
+yarn install --frozen-lockfile
```

Comment:
I was hoping this would install ts-node but it isn't... Do I need to install it in the dockerfile?

Reply:
Not sure what you changed in the last 16 days but this is working fine now